src/docs/asciidoc/en/disruptor.adoc

```diff
@@ -160,7 +160,7 @@ A full memory barrier orders both loads and stores but only on the CPU that exec
 
 Some CPUs have more variants in addition to these three primitives but these three are sufficient to understand the complexities of what is involved.
 In the Java memory model the read and write of a volatile field implements the read and write barriers respectively.
-This was made explicit in the Java Memory Model footnote:jmm[Java Memory Model - http://www.ibm.com/developerworks/library/j-jtp02244/index.html] as defined with the release of Java 5.
+This was made explicit in the Java Memory Model footnote:jmm[Java Memory Model - https://web.archive.org/web/20180119190022/https://www.ibm.com/developerworks/library/j-jtp02244/index.html] as defined with the release of Java 5.
```

Comment:
There are better links to Java Memory Model JSR 133 information: https://jcp.org/en/jsr/detail?id=133